tests/samples/test_target_sqlite.py

```diff
@@ -521,6 +527,10 @@ def test_sqlite_generate_insert_statement(
     assert dml == expected_dml
 
 
+@pytest.mark.xfail(
+    raises=msgspec.DecodeError,
+    reason="Passes when run by itself, Always fails when run with all tests.",
+)
```

Comment:
This smells of dependencies between tests. @BuzzCutNorman can you create an issue for us to address this? 🙏

Reply:
@edgarrmondragon I open an issue as requested, it is #2287.

Reply:
Thanks!